python/ray/train/mosaic/mosaic_trainer.py

```diff
@@ -1,255 +1,30 @@
-import inspect
-import warnings
-from typing import TYPE_CHECKING, Any, Callable, Dict, Optional, Type
+from ray.util.annotations import Deprecated
 
-from composer.loggers.logger_destination import LoggerDestination
-from composer.trainer import Trainer
+_DEPRECATION_MESSAGE = (
+    "`ray.train.mosaic.MosaicTrainer` is deprecated. "
+    "Use `ray.train.torch.TorchTrainer` instead. "
+    "See this issue for a migration example: "
+    "https://github.com/ray-project/ray/issues/42257"
```

Comment:
We won't be able to have a migration example because of the threading issue though, right?

Reply:
Yeah, I was planning to just keep it as "not possible for now", then update the issue later. Or should I just remove the migration link for Mosaic trainer?

Reply:
Oh yeah we can have a GH Issue that says it's not supported right now (the current one points to sklearn)